src/components/MainNavBar/UserProfile.tsx

```diff
@@ -41,30 +42,61 @@ function UserProfile() {
     <div>
       <FormControl data-testid="user-profile-form">
         <Select
```

Comment:
Not part of your story. Just a question- should this menu item be constructed using be` <ul><li>`? If so, I think we can create a new task to address this.

Reply:
My understanding is that html select elements are expected to have option children.
Since this is a non native material ui Select component, I believe it's meant to have MenuItems as children based on the demos
https://mui.com/material-ui/react-select/